third_party/wabt/0003-Add-static-config-file.patch

```diff
@@ -0,0 +1,356 @@
+From 5fcc5f6acaa35422b600efa1c6f42e89012a99a9 Mon Sep 17 00:00:00 2001
+From: Tiziano Santoro <tiziano88@gmail.com>
+Date: Wed, 31 Oct 2018 10:46:27 +0000
+Subject: [PATCH 03/10] Add static config file
+
+---
+ config.h | 337 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
Did you manage to rebuild this using `cmake` as described in https://github.com/project-oak/oak/pull/336#issuecomment-559485683? If so, it may be useful to add those instructions (even at high level) here, so we can rerun it if and when needed.

Reply:
This patch is not my config.h- My config.h was building correctly, but I'm apprehensive to add the instructions as I'm not sure if the `run_examples` failures are related.

Reply:
Ah I see, makes sense, thanks for clarifying.